src/ert/config/analysis_config.py

```diff
@@ -32,23 +41,46 @@ def __init__(
         self._alpha = alpha
         self._std_cutoff = std_cutoff
         self._analysis_iter_config = analysis_iter_config or AnalysisIterConfig()
-        self._update_log_path = update_log_path
+        self._update_log_path = Path(update_log_path)
         self._min_realization = min_realization
 
-        self._analysis_set_var = analysis_set_var or []
-        es_module = AnalysisModule.ens_smoother_module()
-        ies_module = AnalysisModule.iterated_ens_smoother_module()
-        self._modules: Dict[str, AnalysisModule] = {
-            AnalysisMode.ENSEMBLE_SMOOTHER: es_module,
-            AnalysisMode.ITERATED_ENSEMBLE_SMOOTHER: ies_module,
+        options: Dict[str, Dict[str, Any]] = {"STD_ENKF": {}, "IES_ENKF": {}}
+        analysis_set_var = [] if analysis_set_var is None else analysis_set_var
+        inversion_str_map = {
+            "EXACT": "0",
+            "SUBSPACE_EXACT_R": "1",
+            "SUBSPACE_EE_R": "2",
+            "SUBSPACE_RE": "3",
         }
```

Comment:
Consider making this Final, since we know it is not supposed to change?

```
from typing import Final
inversion_str_map : Final= {
             "EXACT": "0",
             "SUBSPACE_EXACT_R": "1",
             "SUBSPACE_EE_R": "2",
             "SUBSPACE_RE": "3",
         }
```